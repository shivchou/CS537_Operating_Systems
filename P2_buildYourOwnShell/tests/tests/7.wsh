alias eh = echo hello
alias eh echo hello
alias eh = 'echo hello'
alias
eh
alias ew = 'echo world'
alias
ew
alias eh = 'echo hello again'
eh
alias e = echo
alias
e e
alias empty = ''
empty echo test
alias empty1 =
empty1 echo test