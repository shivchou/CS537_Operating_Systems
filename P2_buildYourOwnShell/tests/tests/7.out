eh = 'echo hello'
hello
eh = 'echo hello'
ew = 'echo world'
world
hello again
e = 'echo'
eh = 'echo hello again'
ew = 'echo world'
e
test
test
